custom-recipes/nlp-visualization-wordcloud/recipe.py

```diff
@@ -31,12 +32,15 @@
 # Prepare data and count tokens for each subchart
 frequencies = worcloud_visualizer.tokenize_and_count(df)
 
+# Clear output folder's target partition
+output_folder.clear_path(output_partition_path)
```

Comment:
The `clear_path` method is deprecated: https://doc.dataiku.com/dss/latest/python-api/managed_folders.html#dataiku.Folder.clear_path

I would recommend using https://doc.dataiku.com/dss/latest/python-api/managed_folders.html#dataiku.Folder.delete_path or https://doc.dataiku.com/dss/latest/python-api/managed_folders.html#dataiku.Folder.clear_partition

Reply:
corrected sorry